vhost_user_fs/src/passthrough.rs

```diff
@@ -48,7 +48,7 @@ struct InodeData {
 
 struct HandleData {
     inode: Inode,
-    file: Mutex<File>,
+    file: RwLock<File>,
```

Comment:
So, we're actually looking at moving from RwLock to Mutex as it's got better performance.

Reply:
This situation is quite different than the one described in the paper referenced in #777. Here critical section is very large and costly (it even involves at least one syscall!), so the cost of lock management is insignificant in comparison.

In fact, I'd say the conclusions in that paper are only valid if your critical section is extremely small, as it's the case for the code in its benchmark, but that's another story.

Reply:
@slp okay, please leave that comment on #777.